spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -753,8 +756,20 @@ def fixture_record(fixture_name)
     it "skips indexing for Princeton Recap records" do
       expect(@recap_record["recap_notes_display"]).to be nil
     end
+    it "Indexes H - P, if a private SCSB record" do
+      expect(@scsb_private["recap_notes_display"]).to eq ["H - P"]
+    end
+    it "Indexes C - S, if a shared SCSB record" do
+      expect(@scsb_alt_title["recap_notes_display"]).to eq ["C - S"]
+    end
     it "Indexes N - O, if not a private/shared SCSB record" do
       expect(@scsb_nypl["recap_notes_display"]).to eq ["N - O"]
     end
+    it "Indexes H - P, if committed SCSB record" do
+      expect(@scsb_committed["recap_notes_display"]).to eq ["H - C"]
```

Comment:
Can you change the test description to:... H - C ...?

Reply:
Done